test/unit/node/api.js

```diff
@@ -16,7 +16,7 @@ import {suite, test, assert} from '../../unit/unitTestRunner.js';
 
 suite('api.js', function() {
 
-  var api = require('../../src/node/api');
+  var api = require('../../../src/node/api');
```

Comment:
Before we executed the `require()` call from within a nodejs module `/work/traceur-compiler/src/node/System.js`.  Thus the filename `.../../src/node/api` resolves correctly.

To say this another way, our test files were not node modules, just files read by our test app and executed.  Their `require()` calls thus run in the context of the test driver.